pkg/operator/controller/ingress/controller_test.go

```diff
@@ -1648,10 +1669,16 @@ func Test_computeUpdatedInfraFromService(t *testing.T) {
 				},
 			},
 		}
+		ingresses = []corev1.LoadBalancerIngress{
+			{IP: "196.78.125.4"},
+		}
 		ingressesWithMultipleIPs = []corev1.LoadBalancerIngress{
 			{IP: "196.78.125.4"},
 			{IP: "10.10.10.4"},
 		}
+		awsIngresses = []corev1.LoadBalancerIngress{
+			{Hostname: "196.78.125.4"},
```

Comment:
It's probably worth a comment here so devs in the future aren't confused

```suggestion
                        // Hostname is intentionally assigned an IP address for unit testing purposes since net.LookupIP simply returns the provided IP.
			{Hostname: "196.78.125.4"},
```